gatsby-browser.js

```diff
@@ -177,18 +177,18 @@ export const onClientEntry = () => {
 }
 
 const scrollToAnchor = (location) => {
-    // Check for location so build does not fail
-    if (location && location.hash) {
-        window.scrollTo({
-            top: window.innerHeight,
-            behavior: 'smooth',
-        })
+    if(location.hash){
+        const elem = document.querySelector(location.hash);
+        if(elem){
+            elem?.scrollIntoView({ behavior: "smooth" })
+        } else {
+            window.scrollTo(0, 0)
+        }
     }
-
     return true
 }
 
-export const onRouteUpdate = ({ location }) => {
+export const onRouteUpdate = ({location}) => {
     scrollToAnchor(location)
     checkDomain()
 
```

Comment:
@hasan-deriv could you help to format the file for correct spacing , thanks alot 🙏 🌸

Reply:
done